protocol/v2/message/consensus_test.go

```diff
@@ -1,9 +1,10 @@
-package message
+package message_test
```

Comment:
why changed package?   should be in the same package with consensus.go

Reply:
these changes caused one import cycle and are likely to cause more, later we're gonna document about that in some detail here https://blox-ensemble.monday.com/docs/4034094011

what happens is the logging package with the functions that return zap.Fields import type definitions from all over the project. the specific import cycle this caused could be easily fixed by using the `_test` suffix package name for the test files.
 
In this example there is a circular dependency via:
`logging` imports `types`
`types` imports `ssv`
`ssv` imports `logging`

```go
package logging

import (
  "ssv/types"
)

func IDString(id types.ID) zap.Field {
  return ...
}
```

```go
package types

import (
  "ssv/node"
)

type Something struct {
  node node.Node
}
```


```go
package node

import (
  "ssv/logging"
  "ssv/types"
)

func doWork() {
  logger.Info("msg", logging.IDString(someID))
}